bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ImportIntoModelServer.kt

```diff
@@ -83,7 +86,7 @@ abstract class ImportIntoModelServer @Inject constructor(of: ObjectFactory) : De
             client.runWrite(branchRef) { rootNode ->
                 logger.info("Got root node: {}", rootNode)
                 logger.info("Importing...")
-                ModelImporter(rootNode).importFilesAsRootChildren(files)
+                ModelImporter(rootNode, continueOnError.get()).importFilesAsRootChildren(files)
```

Comment:
`continueOnError` only affects the import into the model server this way. For the import into MPS the solution must be changed as well.

Reply:
If you want it only for this direction, I suggest moving the `continueOnError` property to the `ServerTarget` so it's clear for the user.
For the other direction you could also add `continueOnError` for `LocalSource` and then pass the value via a system property in the ant script similar to includedModules. But maybe you should wait for #312. Since those changes will make it easier.

Reply:
I can rebase my branch on that one and we wait for it to be merged.

Reply:
I think I am stuck here. I'd want to implement that also for the direction into MPS. However, due to [#312 being on an outdated state of the sync](https://github.com/modelix/modelix.core/pull/312/files#r1390940876), I can easily built that feature in the current PR. I could log a warning at the moment and implement that once #312 is done.

Reply:
Is that ok with the message for the moment to unblock this?